content/coronavirus_landing_page.yml

```diff
@@ -184,11 +184,21 @@ content:
             - label: Guidance for unpaid carers
               url: /government/collections/coronavirus-covid-19-social-care-guidance#guidance-for-unpaid-carers
             - label: Parking passes for healthcare workers and volunteers
-              url: /government/publications/covid-19-health-care-and-volunteer-workers-parking-pass-and-concessions
+              url: /government/publications/covid-19-health-care-and-volunteer-workers-parking-pass-and-concession
         - title: Testing
           list:
             - label: Testing for frontline workers
               url: /guidance/coronavirus-covid-19-getting-tested
+    - title: Support if someone dies
+      sub_sections:
+        - title:
+          list:
+            - label: What to do when someone dies: step-by-step
```

Comment:
You'll need to wrap the text on this line in quotes so that it looks like this:

`- label: "What to do when someone dies: step-by-step"`

It's because the text has a colon in it with a space afterwards, so the Yaml processor thinks it means something special

Reply:
I can't seem to get back into this to add the quote marks!